packages/stryker/src/StrykerCli.ts

```diff
@@ -0,0 +1,93 @@
+import * as program from 'commander';
+import { CONFIG_SYNTAX_HELP } from './ConfigReader';
+import Stryker from './Stryker';
+import StrykerInitializer from './initializer/StrykerInitializer';
+import * as log4js from 'log4js';
+
+const log = log4js.getLogger('stryker-cli');
+let command: string = '', strykerConfig: string | null = null;
```

Comment:
We can move these two variables inside of the class so they become properties :)